specification/authorization/resource-manager/Microsoft.Authorization/2017-10-01-preview/authorization-RACalls.json

```diff
@@ -0,0 +1,612 @@
+  "definitions": {
+    "RoleAssignmentFilter": {
+      "properties": {
+        "principalId": {
+          "type": "string",
+          "description": "Returns role assignment of the specific principal."
+        },
+        "canDelegate":{
+          "type": "boolean",
+          "description": "The Delegation flag for the roleassignment"
+        }
+      },
+      "description": "Role Assignments filter"
+    },
+    "RoleAssignmentPropertiesWithScope": {
+      "properties": {
+        "scope": {
+          "type": "string",
+          "description": "The role assignment scope."
+        },
+        "roleDefinitionId": {
+          "type": "string",
+          "description": "The role definition ID."
+        },
+        "principalId": {
+          "type": "string",
+          "description": "The principal ID."
+        },
+        "canDelegate":{
+          "type": "boolean",
+          "description": "The Delegation flag for the roleassignment"
+        }
+      },
+      "description": "Role assignment properties with scope."
+    },
+    "RoleAssignment": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly":false,
+          "description": "The role assignment ID."
+        },
+        "name": {
+          "type": "string",
+          "readOnly":false,
+          "description": "The role assignment name."
+        },
+        "type": {
+          "type": "string",
+          "readOnly":false,
+          "description": "The role assignment type."
+        },
+        "properties": {
+          "$ref": "#/definitions/RoleAssignmentPropertiesWithScope",
+          "description": "Role assignment properties."
+        }
+      },
+      "description": "Role Assignments"
+    },
+    "RoleAssignmentListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RoleAssignment"
+          },
+          "description": "Role assignment list."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to use for getting the next set of results."
+        }
+      },
+      "description": "Role assignment list operation result."
+    },
+    "RoleAssignmentProperties": {
+      "properties": {
+        "roleDefinitionId": {
+          "type": "string",
+          "description": "The role definition ID used in the role assignment."
+        },
+        "principalId": {
+          "type": "string",
+          "description": "The principal ID assigned to the role. This maps to the ID inside the Active Directory. It can point to a user, service principal, or security group."
+        },
+        "canDelegate": {
+          "type": "boolean",
+          "description": "The delgation flag used for creating a role assignment"
+        }
+      },
+      "description": "Role assignment properties."
+    },
+    "RoleAssignmentCreateParameters": {
+      "properties": {
+        "properties": {
```

Comment:
>properties [](start = 9, length = 10)

This property can also benefit from being [flattened](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-client-flatten)

Reply:
Added the x-ms-client-flatten in all the places the linter tool showed as warning